pandas/tests/indexes/multi/test_constructor.py

```diff
@@ -610,11 +610,10 @@ def test_create_index_existing_name(idx):
             ],
             dtype="object",
         ),
-        names=["foo", "bar"],
     )
     tm.assert_index_equal(result, expected)
 
-    result = pd.Index(index, names=["A", "B"])
+    result = pd.Index(index, name="A")
```

Comment:
Both result an dexpected are normal ``Index`` and instantiating with a ``names`` parameter should fail.